source/common/common/utility.cc

```diff
@@ -67,6 +71,65 @@ bool StringUtil::atoul(const char* str, uint64_t& out, int base) {
   }
 }
 
+absl::string_view StringUtil::ltrim(absl::string_view source) {
+  const absl::string_view::size_type pos = source.find_first_not_of(WhitespaceChars);
+  if (pos != absl::string_view::npos) {
+    source.remove_prefix(pos);
+  } else {
+    source.remove_prefix(source.size());
+  }
+  return source;
+}
+
+absl::string_view StringUtil::rtrim(absl::string_view source) {
+  const absl::string_view::size_type pos = source.find_last_not_of(WhitespaceChars);
+  if (pos != absl::string_view::npos) {
+    source.remove_suffix(source.size() - pos - 1);
+  } else {
+    source.remove_suffix(source.size());
+  }
+  return source;
+}
+
+absl::string_view StringUtil::trim(absl::string_view source) { return ltrim(rtrim(source)); }
+
+bool StringUtil::findToken(absl::string_view source, absl::string_view delimiters,
+                           absl::string_view key_token, bool trim_whitespace) {
+  const std::vector<absl::string_view> tokens = splitToken(source, delimiters, false);
```

Comment:
OK I'll let you make the final call, but IMO you could just do:
```
  const std::vector<absl::string_view>tokens = splitToken(source, delimeters, trim_whitespace);
  return std::find(tokens.begin(), tokens.end(), key_token) != tokens.end();
```
which changes the definition of ("", ",", "") to give you a 'false'.  If you really like spelling out the code like this I can accept it, but anyone reading will wonder why this is so complicated.

Reply:
Sounds good :)